Bluetooth: Removed _gatt_ infix for BAS, DIS and HRS (#28050)
zephyrproject-rtos/zephyr: doc/releases/release-notes-2.4.rst

  const qualifier loss on ISRs, all ISRs now take a ``const *void`` as a
  paremeter as well.

> * The ``_gatt_``/``_GATT_`` infix for HRS, DIS and BAS was removed for the API

Reviewer: ```suggestion
* The `_gatt_` and `_GATT_` infixes have been removed for the HRS, DIS and BAS APIs and Kconfig functions.
```

Author: Applied your suggestion. I was a bit unsure of `Kconfig functions` was the right terminology (rather than `Kconfig options`, so I went with your suggestion. Please let me know if `options` is the better choice.

>   and tha Kconfig options.

Reviewer: ```suggestion
```

> * The ``_gatt_`` and ``_GATT_`` infixex have been removed for the HRS, DIS
>   and BAS APIs and the Kconfig functions.

Reviewer 2: ```suggestion
* The ``_gatt_`` and ``_GATT_`` infixes have been removed for the HRS, DIS
  and BAS APIs and the Kconfig options.
```

Author: Nice catch with the typo (I guess my typo checker broke since it didn't catch that).
